Elevel/Application/Features/AudioFeatures/GetAuditionListQuery.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using AutoMapper;
+using Elevel.Application.Interfaces;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Elevel.Application.Pagination;
+
+namespace Elevel.Application.Features.AudioFeatures
+{
+    public class GetAuditionListQuery
+    {
+        public class Request : PagedQueryBase, IRequest<Response>
+        {
+            public Guid Id { get; set; }
+            public string AudioFilePath { get; set; }
+        }
+        public class Handler: IRequestHandler<Request, Response>
+        {
+            private readonly IApplicationDbContext _context;
+            private readonly IMapper _mapper;
+            public Handler(IApplicationDbContext context, IMapper mapper)
+            {
+                _context = context;
+                _mapper = mapper;
+            }
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var auditionList = _context.Auditions.AsNoTracking()
+                    .Include(c => c.Id)
+                    .Include(c => c.AudioFilePath)
```

Comment:
https://www.entityframeworktutorial.net/efcore/querying-in-ef-core.aspx read this article